case_utils/case_validate/__init__.py

```diff
@@ -259,8 +371,9 @@ def main() -> None:
     if args.format != "human":
         validator_kwargs["serialize_report_graph"] = args.format
 
-    validate_result: Tuple[bool, Union[Exception, bytes, str, rdflib.Graph], str]
-    validate_result = pyshacl.validate(
+    validate_result: Tuple[
+        bool, Union[Exception, bytes, str, rdflib.Graph], str
+    ] = pyshacl.validate(
```

Comment:
I think this call needs to be replaced with the `validate()` method this PR is adding to this file, but only after logistics related to #123 are settled.

Reply:
I believe this is addressed in `15f00c9`.